google/cloud/storage/tests/thread_integration_test.cc

```diff
@@ -27,35 +27,13 @@ namespace cloud {
 namespace storage {
 inline namespace STORAGE_CLIENT_NS {
 namespace {
-/// Store the project and instance captured from the command-line arguments.
-class ThreadTestEnvironment : public ::testing::Environment {
- public:
-  ThreadTestEnvironment(std::string project, std::string location) {
-    project_id_ = std::move(project);
-    location_ = std::move(location);
-  }
-
-  static std::string const& project_id() { return project_id_; }
-  static std::string const& location() { return location_; }
-
- private:
-  static std::string project_id_;
-  static std::string location_;
-};
+using ObjectNameList = std::vector<std::string>;
 
-std::string ThreadTestEnvironment::project_id_;
-std::string ThreadTestEnvironment::location_;
+char const* flag_project_id;
```

Comment:
(optional) Consider if you think it'd be worthwhile to add a comment to these little blocks of namespace-scope varaibles like "// initialized in main() below" or something.

Reply:
Done.